canton/BUILD.bazel

```diff
@@ -880,7 +880,7 @@ scala_library(
     ],
     unused_dependency_checker_mode = "error",
     deps = [
-        ":community_participant_admin-workflows-with-vacuuming_java.jar",
+        ":community_participant_admin-workflows-with-vacuuming_java",
```

Comment:
Don't we need to apply similar change on the canton-3x folder?

Reply:
@remyhaemmerle-da

Reply:
@mziolekda where is this folder located?

Reply:
Thanks for catching this @mziolekda! Yes, I just checked:

```
$ git checkout atriantafyllos/convert-conformace-tests-to-use-java-bindings
$ git rebase main
$ ci/build-canton-3x.sh
[...]
ERROR: /home/polux/projects/daml/canton-3x/BUILD.bazel:865:14: in deps attribute of scala_library rule //canton-3x:community_participant: target '//canton-3x:community_participant_admin-workflows-with-vacuuming_java.jar' does not exist
ERROR: /home/polux/projects/daml/canton-3x/BUILD.bazel:865:14: Analysis of target '//canton-3x:community_participant' failed
```
So you need to make the same change to `canton3/BUILD.bazel`.

Beware that if you run this script, it will create leftovers (it is meant to be run by the CI), namely:
- canton-3x/community
- canton-3x/daml-common-staging
- changes to .bazelignore

Reply:
17b1c2add6cfed9a6feb183b6d5d25dfd0b9897c